packages/teams-js/src/private/copilot/customTelemetry.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @beta
+ * @hidden
+ * User information required by specific apps
+ * @internal
+ * Limited to Microsoft-internal use
+ * @module
+ */
+
+import { callFunctionInHost } from '../../internal/communication';
+import { ensureInitialized } from '../../internal/internalAPIs';
+import { ApiName, ApiVersionNumber, getApiVersionTag, getLogger } from '../../internal/telemetry';
+import { getCurrentTimestamp } from '../../internal/utils';
+import { runtime } from '../../public/runtime';
+
+const copilotTelemetryVersionNumber: ApiVersionNumber = ApiVersionNumber.V_2;
+const copilotLogger = getLogger('copilot');
+
+export enum Stage {
+  STAGE_E = 'E',
```

Comment:
Adding some comments for StageE? 

Also I am not sure if StageE is good naming... it looks only Copilot folks will know what StageE is .. is it possible to use some generic naming like "NotifyUIUpdateFromAppToHost", or something that reflect StageE more close?